.github/workflows/test.yml

```diff
@@ -49,10 +49,81 @@ jobs:
           DANGER_GITHUB_API_TOKEN: ${{ secrets.DANGER_GITHUB_API_TOKEN }}
         run: bundle exec danger
 
+  ios_test:
+    name: Running tests in iOS
+    runs-on: macos-latest
+    defaults:
+      run:
+        working-directory: ./sample/ios
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - uses: actions/cache@v3
```

Comment:
You can use this one to reduce steps (cache, bundle install): https://github.com/ruby/setup-ruby#single-job

Reply:
@suho I reverted these changes as it made the generating step stop working 
I will further check this later